docs/sources/datasources/tempo.md

```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 ## Query traces
 
 You can query and display traces from Tempo via [Explore]({{< relref "../explore/_index.md" >}}).
-To query a particular trace, insert its trace ID into the query text input.
+You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.
 
-{{< figure src="/static/img/docs/v73/tempo-query-editor.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor" >}}
+{{< figure src="/static/img/docs/tempo/query-editor-search.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor showing the search tab" >}}
+
+To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.
+
+{{< figure src="/src/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}
```

Comment:
```suggestion
{{< figure src="/static/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}
```

Assuming you want the image from https://github.com/grafana/website/pull/4510.

Reply:
yes thank you, I missed this one